files/ol3-geocoder/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "ol3-geocoder",
+  "repo": "jonataswalker/ol3-geocoder",
+  "files": {
+    "include": ["ol3-geocoder.js", "ol3-geocoder-debug.js", "ol3-geocoder.css"]
```

Comment:
I dont think `ol3-geocoder-debug.js` is required in production. Can you remove it?